core/src/test/resources/org.apache.iceberg.view/ViewMetadataInvalidCurrentSchema.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
The path for all the view json files in resources seems to be off should be core/src/test/resources/org/apache/iceberg/view/ViewMetadataInvalidCurrentSchema.json

same with others.

Reply:
good catch, somehow IntelliJ couldn't rename this to a folder structure. Should be fixed now